### PR TITLE
Fix Files(IPFS) module status link and download link

### PR DESCRIPTION
**Pull request by XueMoMo**
1. The old download link is no longer available, so we should use cf-ipfs.com
2. Fix the status link to link to specific file information

### CODE REVIEW

**Comment by itsbirdo** [comment]
@TarikGul are you able to look at this please sir ?

File: packages/page-files/src/CrustFiles.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ const shortStr = (name: string, count = 6): string => {
 };
 
 function createUrl (f: SaveFile) {
-  const endpoint = f.UpEndpoint || 'https://ipfs.io';
+  const endpoint = 'https://cf-ipfs.com';
```
Comment (TarikGul) [inline packages/page-files/src/CrustFiles.tsx:46]: Why do we get rid of `f.UpEdnpoint`?

**Review by TarikGul — CHANGES_REQUESTED** [review]
Just one question. Otherwise looks good.

File: packages/page-files/src/CrustFiles.tsx (hunk 1 above)
Comment (XueMoMo) [reply packages/page-files/src/CrustFiles.tsx:46]: Because currently UpEndpoint is only used for uploading files and cannot be used to access files.

**Review by TarikGul — APPROVED** [review]
Apologies on the delay. LGTM

**Pull request merged**